docs/snippets/ov_model_snippets.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+#! [import]
+import openvino.runtime as ov
+#! [import]
+import openvino.runtime.passes as passes
+
+# ! [ov:create_simple_model]
+def create_simple_model():
+    # This example shows how to create ov::Function
+    #
+    # Parameter--->Multiply--->Add--->Result
+    #    Constant---'          /
+    #              Constant---'
+    data = ov.opset8.parameter(ov.Shape([3, 1, 2]), ov.Type.f32)
+    mul_constant = ov.opset8.constant(ov.Type.f32, ov.Shape({1}), [1.5])
+    mul = ov.opset8.multiply(data, mul_constant)
+    add_constant = ov.opset8.constant(ov.Type.f32, ov.Shape({1}), [0.5])
```

Comment:
```suggestion
    data = ov.opset8.parameter([3, 1, 2], ov.Type.f32)
    mul_constant = ov.opset8.constant([1.5], ov.Type.f32)
    mul = ov.opset8.multiply(data, mul_constant)
    add_constant = ov.opset8.constant([0.5], ov.Type.f32)
```
First there is no need to use `ov.Shape` in wrapped constructors -- thsi gives more "pythonic feel".

Second: constants from opsets can be replaced as `openvino.runtime.op.Constant(ov.Type.f32, ov.Shape({1}), [1.5])` to match cpp approach, the code won't work if Type is first argument. Additionally, in opsets constant wrapper infer shape from passed data. Same story for the rest of examples.